lapce-ui/src/svg.rs

```diff
@@ -49,15 +49,33 @@ pub fn get_svg(name: &'static str) -> Option<Svg> {
 }
 
 pub fn file_svg(path: &Path) -> Svg {
-    let file_type = if path.file_name().and_then(OsStr::to_str) == Some("LICENSE") {
+    
+    // catch if path.to_str() is Some without calling it twice
+    let path_str = path.to_str();
+    if path_str == None {
```

Comment:
This isn't really necessary. `path_str.expect("Missing path");` should be enough in place of the unwrap below.